stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messages/MessageText.kt

```diff
@@ -140,3 +150,49 @@ private fun buildAnnotatedMessageText(message: Message): AnnotatedString {
         }
     }
 }
+
+/**
+ * A spin-off of a Foundation component that allows calling long press handlers.
+ * Contains only one additional parameter.
+ *
+ * @param onLongPress Handler called on long press.
```

Comment:
I did not want to simply copy paste the docs from the original implementation in 'foundation' or reinvent them.

If you think they are necessary I will do so.

Reply:
That's totally fine, people can browse the foundation impl